climpred/utils.py

```diff
@@ -4,6 +4,9 @@
 import numpy as np
 import xarray as xr
 
+# KVP added
```

Comment:
This is fine -- just add to reqs as mentioned. And move the pandas import below numpy above.